internal/provisioner/fluentbit.go

```diff
@@ -0,0 +1,50 @@
+package provisioner
+
+import (
+	"fmt"
+
+	"github.com/mattermost/mattermost-cloud/internal/tools/kops"
+)
+
+type fluentbit struct {
+	provisioner *KopsProvisioner
+	kops        *kops.Cmd
+}
+
+func NewFluentbitHandle(provisioner *KopsProvisioner, kops *kops.Cmd) (*fluentbit, error) {
+
+	if provisioner == nil {
+		return nil, fmt.Errorf("Cannot create a connection to Prometheus is the provisioner provided is nil")
```

Comment:
when I copy and paste error messages I need to remember to change the name of the service :P